index.html

```diff
@@ -57,10 +60,313 @@ <h1>I'm Salma <br> Glad to see you!</h1>
                 <a href="" target="_blank" rel="noopener noreferrer">
                     <img class="s-m-icon" src="images/Vector (3).svg" alt="social-icon">
                 </a>
-               </li>
-                
+               </li>  
             </ul>
     </section> 
+
+<!--Work Cards Section-->
+
+    <section class="work-card">
+        <img src="images/Snapshoot Portfolio.png" alt="Snapshoot">
+        <h1>Tonic</h1>
+        
+        <article class="wrap-frame">
+            <div class="frame canopy"><p>CANOPY</p></div>
+            <div class="frame">
+                <img src="images/Counter.png" alt="">
+            </div>
+            <div class="frame">
+                <p>Back End Dev</p>
+            </div>
+            <div class="frame">
+                <img src="images/Counter.png" alt="">
+            </div>
+            <div class="frame">
+                <p>2015</p>
+            </div>
+            </article>
+
+            <div class="par">
+            <p> A daily selection of privately
+                 personalized reads;<br> no accounts or
+                 sign-ups required.</p>
+            </div>
+            <div class="lang-button">
+                <ul>
+                    <li>
+                        <a href="#">Html</a>
+                    </li>
+                    <li>
+                        <a href="#">css</a>
+                    </li>
+                    <li>
+                        <a href="#">Javascript</a>
+                    </li>
+                </ul>
+            </div>
+
+            <button type="button" class="btn">See Project</button>
+    </section> 
+
+
+    <section class="work-card">
+        <img src="images/Snapshoot Portfolio (2).png" alt="Snapshoot">
+        <h1>Tonic</h1>
+        
+        <article class="wrap-frame">
```

Comment:
Your `<article>` tag doesn't have a closing tag, kindly add a closing tag as this is semantically appropriate.🤗